esp/services/ws_smc/ws_smcService.cpp

```diff
@@ -2226,6 +2226,8 @@ bool CWsSMCEx::onRoxieControlCmd(IEspContext &context, IEspRoxieControlCmdReques
     ISmartSocketFactory *conn = roxieConnMap.getValue(target);
     if (!conn)
         throw makeStringExceptionV(ECLWATCH_CANNOT_GET_ENV_INFO, "roxie target cluster not mapped: %s", target);
+    if (!checkK8sEndpoints(target))
+        throw makeStringExceptionV(ECLWATCH_CANNOT_GET_ENV_INFO, "roxie target cluster has no active endpoints: %s", target);
```

Comment:
could possibly have a more user friendly error message - e.g. no running instances instead of no active endpoints?